src/components/interfaces/MOBILE_API.xml

```diff
@@ -7475,6 +7475,13 @@
         <param name="state" type="DriverDistractionState" mandatory="true">
             <description>Current State of Driver Distraction</description>
         </param>
+        <param name="lockScreenDismissalEnabled" type="Boolean" mandatory="false">
```

Comment:
add `since="5.2"`

Reply:
@jacobkeeler Thanks. Fixed in c810f4cced24c920e865a5b3d29b494d875bf781